k8s/core/configmap/configmap_lock_v2.go

```diff
@@ -174,13 +175,17 @@ func (c *configMap) IsKeyLocked(key string) (bool, string, error) {
 	}
 
 	if owner, ok := lockIDs[key]; ok {
-		// Someone owns the lock: let's check the expiration and the owner
+		// Existing key is unlocked if
+		//   1. Lock has expired; or
+		//   2. Lock owned by itself but goroutine refreshing expiration time doesn't exist
 		expiration := lockExpirations[key]
 		if time.Now().After(expiration) {
-			// Lock is expired
```

Comment:
Why remove the comment?

Reply:
The explanation is moved a few lines above